index.html

```diff
@@ -141,6 +141,9 @@ <h3>File Sharing</h3>
                    be supported by servers as well as user agents.
    N15             It must be possible to support data exchange
                    in a worker.
+   N29             The application must be able to request user consent
```

Comment:
"Request user consent" is an implementation, not a requirement. Please rephrase what the requirement is.

Reply:
Is it just the word "request"? What about replacing it with "obtain"?